idp_mgmt_docs.adoc

```diff
@@ -1,6 +1,11 @@
-
 [#idp-mgmt-docs]
 = identity configuration management for Kubernetes documentation (Technology Preview)
+:version-label: version
+:chapter-label:
+:toc: left
+:toclevels: 0
+:toc-title: Table of Contents
+v0.1.0: IDP
```

Comment:
IDP isn't an approved acronym. This should be removed.

Reply:
identity configuration management for Kubernetes